src/main/java/io/github/hielkemaps/racecommand/race/Race.java

```diff
@@ -46,18 +52,44 @@ public Race(UUID owner, String name) {
 
     public void start() {
         place = 1;
+        huntedPlace = 1;
         finishedPlayers = new ArrayList<>();
+        huntedPlayers = new ArrayList<>();
         results = new ArrayList<>();
         isStarting = true;
 
         AtomicInteger seconds = new AtomicInteger(countDown);
 
         sendMessage(Main.PREFIX + "Starting race...");
 
+        for(UUID uuid : players) {
+            Player player = Bukkit.getPlayer(uuid);
+            player.removeScoreboardTag("hunter");
+            player.removeScoreboardTag("runner");
+        }
+
+        //Set hunter for manhunt races
+        if(variantType() == "manhunt") {
+            int hunterRandomizer = (int) Math.random() * players.size();
```

Comment:
Our static analysis engine detects that your code is problematic. According to the [Correctness (CORRECTNESS)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#correctness-correctness), [RV: Random value from 0 to 1 is coerced to the integer 0 (RV_01_TO_INT)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#rv-random-value-from-0-to-1-is-coerced-to-the-integer-0-rv-01-to-int),

A random value from 0 to 1 is being coerced to the integer value 0. You probably want to multiply the random value by something else before coercing it to an integer, or use the `Random.nextInt(n)` method.

Here, consider to use `(int) (Math.random() * players.size())`.